sky/clouds/gcp.py

```diff
@@ -583,7 +583,10 @@ def _get_feasible_launchable_resources(
                                         resources.disk_tier)
             if not ok:
                 return resources_utils.FeasibleResources([], [], None)
-            return resources_utils.FeasibleResources([resources], [], None)
+
+            if resources.instance_type not in service_catalog.gcp_catalog.GCP_ACC_INSTANCE_TYPES:
+                # General CPU instance types without accelerators
+                return resources_utils.FeasibleResources([resources], [], None)
```

Comment:
Do we need this? It seems the handling below is not required for instance type that has GPUs attached?